proto/poktroll/shared/params.proto

```diff
@@ -12,4 +12,5 @@ message Params {
   option (gogoproto.equal) = true;
 
   uint64 num_blocks_per_session = 1 [(gogoproto.jsontag) = "num_blocks_per_session"];
+  uint64 claim_window_open_offset_blocks = 2 [(gogoproto.jsontag) = "claim_window_open_offset_blocks"];
```

Comment:
#PUC on what this is, what its for, how its used.

(e.g. number of blocks after session ends ..)

Reply:
Shouldn't we start using this in https://github.com/pokt-network/poktroll/blob/issues/516/feat/claim-window-open-offset-blocks/pkg/relayer/session/claim.go#L94

Reply:
❓

Reply:
@red-0ne It's in the pipeline: https://github.com/pokt-network/poktroll/pull/565/files#diff-78aa6b89c55aadb85f27b5f9af9cd4b4264dc1389e64b9ada24fae6f99cc9791R93. This PR is just adding the `ClaimWindowOpenOffsetBlocks` param.